syntax = "proto3";

service Main {
  rpc SendMessage(Message) returns(Empty);
  rpc GetMessages(Empty) returns (Message);
}
message Message {
   string nickname = 1;
   string datetime = 2;
   bytes encrypted_text = 3;
}

message Empty {

}